src/main.py

```diff
@@ -17,21 +18,52 @@ def main():
     cap = cv.VideoCapture(0)
     detector = HandDetector()
     monitor = get_monitors()
+    m_width = monitor[0].width
+    m_height = monitor[0].height
 
     # генерация клавиш и пианино
-    # piano = Piano(100,100,200,200)
+    piano = Piano(0, 0, int(m_width/1.5), int(m_height/3))
+    spath = os.path.abspath('')[:-7] + 'sounds\sound_7'
```

Comment:
incorrect path
possible fix:
`spath = os.path.abspath('') + '\sounds\sound_7'`

Reply:
you forgot `\`